content/articles/flutter_speech_recognition_using_alan_voice/index.md

```diff
@@ -0,0 +1,228 @@
+
+
+![Hero](/flutter_speech_recognition_using_alan_voice/hero.png)
+
+Speech recognition is not a new technology, it has been around for a couple of years and has led to the development of amazing features in modern applications being created. Some of these features include voice assistance and speech to text. In this tutorial we will see how to create an application that performs actions based on commands gotten from recognising the speech of the user.
+
+
+### Introduction
+Alan voice is an Artificial intelligent powered platform that uses conversational voice to create powerful voice assistance applications. When creating your applications that support speech recognition, changes to the setups are usually made on version releases and updates you make to the initial app build. This is inefficient and hectic to implement. It would be better if the underlying technology that contains the basics for the speech recognition to function well exists and sits locally and whenever any updates need to be made to the functions of our commands, it will be made on a much simpler and easier interface that is easily configurable. With alan voice plugin, we can set up basic functionality in our application that can communicate with users using speech recognition to perform complex function setup on an easy configurable interface provided to us by alan voice. 
+
+To achieve this we will first set up our flutter application, then we will link it to alan voice by integrating alan voice plugin into the application. Next we will test our speech recognition application. Finally we will create/edit/update our commands without updating our application.
+
+
+### Prerequisite
```

Comment:
Please put all your prerequisite in an unordered list format, also include other prerequisites necessary for the reader to follow along such as a preferable IDE, Flutter SDK version, Alan SDK etc.